tests/core/mempool/test_mempool_manager.py

```diff
@@ -534,3 +535,93 @@ async def test_get_items_not_in_filter() -> None:
     sb2_and_3_filter = PyBIP158([bytearray(sb2_name), bytearray(sb3_name)])
     result = mempool_manager.get_items_not_in_filter(sb2_and_3_filter)
     assert result == [sb1]
+
+
+def always(_: bytes32) -> bool:
+    return True
+
+
+def never(_: bytes32) -> bool:
+    return False
+
+
+class TestFor(Enum):
+    MAX_COST = 0
+    MAX_FEE = 1
+    NORMAL_PATH = 2
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("test_for", [t for t in TestFor])
+@pytest.mark.parametrize("inclusion_filter", [always, never])
```

Comment:
I don't think testing the inclusion filter is important. We only use it in one test, never in production